crates/uv/src/commands/project/run.rs

```diff
@@ -63,7 +66,11 @@ pub(crate) async fn run(
     // Determine whether the command to execute is a PEP 723 script.
     let script_interpreter = if let RunCommand::Python(target, _) = &command {
         if let Some(metadata) = uv_scripts::read_pep723_metadata(&target).await? {
-            debug!("Found PEP 723 script at: {}", target.display());
+            writeln!(
+                printer.stderr(),
+                "Running PEP 723 script at: {}",
```

Comment:
Can we refer to "inline script metadata" instead of the PEP? I think it'd be good to normalize the term and discuss things as the user-facing concept.

Reply:
Ok